web/src/components/njwds-extended/HelpButton.tsx

```diff
@@ -9,11 +9,11 @@ export const HelpButton = (): ReactElement => {
   return (
     <div className="flex fac margin-top-1 mobile-lg:margin-top-0 mobile-lg:margin-left-auto mobile-lg:margin-right-105 width-full mobile-lg:width-auto">
       <UnStyledButton
-        style={"unstyled"}
+        style={"transparentBgColor"}
         className={"text-accent-cool-darker fjc padding-0"}
-        textBold={true}
-        intercomButton
-        noRightMargin={true}
+        isTextBold
+        isIntercomEnabled
+        isRightMarginRemoved={true}
```

Comment:
[Dust] A bit inconsistent here to say `isRightMarginRemoved={true}` instead of just `isRightMarginRemoved` but I imagine you did this because its more readable?

Reply:
Either scenario would work here, I noticed that my IDE automatically creates the curly braces so I don't go to the process of deleting them sometimes. I'll look into a setting to disable this on my IDE. I think we using either one interchangeably, the fact that you justify in the prop implies that it is true.

Reply:
Added as a techtro discussion item